### PR TITLE
OHRM5X-1151: Define i18n - admin module language strings

### PR DESCRIPTION
**Pull request by SharukaNP**

### CODE REVIEW

File: symfony/client/src/orangehrmAdminPlugin/pages/systemUser/EditSystemUser.vue [hunk 1]
```diff
@@ -155,17 +155,17 @@ export default {
           shouldNotExceedCharLength(40),
           promiseDebounce(this.validateUserName, 500),
         ],
-        role: [v => (!!v && v.length != 0) || 'Required'],
+        role: [required],
```
Comment (SharukaNP) [inline symfony/client/src/orangehrmAdminPlugin/pages/systemUser/EditSystemUser.vue:158]: @https://github.com/Super-Chama I've changed this validation into the required validation in rules.ts and based on testing on the view it gave the error message when kept empty. Hope that's ok

Comment (RajithaKumara) [reply symfony/client/src/orangehrmAdminPlugin/pages/systemUser/EditSystemUser.vue:158]: Yeah. fine we have to track these changes. may be screen wise

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    unitId: end_time_should_be_after_start_time
+  -
+    value: 'Attachment Size Exceeded'
+    unitId: attachment_size_exceeded
+  -
+    value: 'File type not allowed'
+    unitId: file_type_not_allowed
+  -
+    value: 'Expected format: admin@example.com'
+    unitId: expected_format_not_matched
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:184]: changed

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    unitId: add
+  -
+    value: 'Phone'
+    unitId: phone
+  -
+    value: 'State/Province'
+    unitId: state/province
+  -
+    value: 'Zip/Postal Code'
+    unitId: zip/postal_code
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:28]: sure

File: installer/upgrader/Migrations/V5/adminLangString.yaml [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
-    unitId: add_job_titles
+    value: Add Job Title
+    unitId: add_job_title
+  -
+    value: Edit Job Titles
+    unitId: edit_job_title
   -
     value: Job Title
-    unitId: job_title
+    unitId: job_titles
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/adminLangString.yaml:10]: here the mistake was in the value I adjusted it

**Comment by RajithaKumara** [comment]
please run `yarn lint` inside `symfony/client` directory, and commit changes

**Review by RajithaKumara — APPROVED** [review]

**Comment by SharukaNP** [comment]
How to test:
1. follow this to set up the localization in 5x https://github.com/orangehrm/orangehrm/pull/1115#issuecomment-1028677535
2. add the .xml file into orangehrm/tree/installer/upgrader/Migrations/V5
[translation bg.zip](https://github.com/orangehrm/orangehrm/files/8176123/translation.bg.zip)
3.run php installer/upgrader/langStringHandler.php inside PHP container
4.run php installer/upgrader/runTranslations.php inside PHP container

**Pull request merged**